lib/fizzy/execute.cpp

```diff
@@ -1199,7 +1199,7 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
                 goto trap;
             const auto lhs = stack.top().as<int64_t>();
             if (lhs == std::numeric_limits<int64_t>::min() && rhs == -1)
-                stack.top() = 0;
+                stack.top().i64 = 0;  // FIXME: Review all cases like this.
```

Comment:
I don't insist it's much better, but this way it would look more like you're just returning 64-bit value, without assuming internal representation of stack item
```suggestion
                stack.top() = uint64_t{0};  // FIXME: Review all cases like this.
```

Reply:
I'd agree this is cleaner, since the `rem()` case below is defined as `T rem(T a, T b)`, so both would be assigning a `int64_t` to `top()`.

Reply:
In that sense also I do not get the FIXME, having `top() = uint64_t{0}` seems to be the correct approach.

Reply:
Done.